### ISSUE TITLE
Remove preprocessing data loaders?

### ISSUE THREAD

**Issue by rpreen** [AI-SDC/SACRO-ML#295]
Should the preprocessing data loaders be part of the `aisdc` package? It doesn't seem likely that many people will want to use them and they are more suited to going in a repository dedicated to running experiments.

**Comment by jim-smith** [comment]
@rpreen didn't you do this already?

**Comment by rpreen** [comment]
Nope

**Comment by albacrespi** [comment]
I agree @rpreen , it doesn't seem anything is really using or need to use them and it's more appropiate to have them in the SACRO-Research repo where the experiments are.
